### DISCUSSION TITLE
Windows Prisma Engine detected as virus

### DISCUSSION THREAD

**Discussion by aferreiro00** [prisma/prisma#21690] [Q&A]
### Question

I use FortiClient with the Antivirus integrated. Everytime I try to run generate prisma tries to download migrate-engine-windows.exe and the antivirus detects it as a virus and puts it in quarantine.

I use Prisma v4.16.2. Is there any chance of this getting corrected? I use a company PC and I cannot disable the antivirus, nor update Prisma because of the project requirements.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @aferreiro00 👋 

Thank you for raising this question.

is it possible to set up an exclusion or whitelist for the Prisma application or its relevant files in FortiClient to prevent false positives?

**Comment by aferreiro00** [comment]
Sadly, I cannot do that. I use a company PC and most of the options are disabled because of the lack of rights. I didn't see any whitelist option, anyway. And I cannot disable the real-time protection.

I also asked for help to my company's support team, but I still have no replies.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
